en/04_Changelogs/5.2.0.md

```diff
@@ -7,17 +7,42 @@ title: 5.2.0 (unreleased)
 ## Overview
 
 - [Features and enhancements](#features-and-enhancements)
+  - [New ORM features](#new-orm-features)
   - [Other new features](#other-new-features)
 - [API changes](#api-changes)
 - [Bug fixes](#bug-fixes)
 
 ## Features and enhancements
 
+### New ORM features {#new-orm-features}
+
+This release comes jampacked with new ORM features, granting you access to some new abstractions for more powerful and efficient queries.
+
+#### Common Table Expressions (aka the WITH clause) {#orm-with-clause}
+
+Something about the new [`DataQuery::with()`](api:SilverStripe\ORM\DataQuery::with()) and [`SQLSelect::addWith()`](api:SilverStripe\ORM\Queries\SQLSelect::addWith()) methods
+
+#### UNION clause {#orm-union-clause}
+
+Something about the new [`DataQuery::union()`](api:SilverStripe\ORM\DataQuery::union()) and [`SQLSelect::addUnion()`](api:SilverStripe\ORM\Queries\SQLSelect::addUnion()) methods
+
+#### RIGHT JOIN clause {#orm-right-join}
+
+Something about the new [`DataQuery::rightJoin()`](api:SilverStripe\ORM\DataQuery::rightJoin()) and [`SQLConditionalExpression::rightJoin()`](api:SilverStripe\ORM\Queries\SQLConditionalExpression::addRightJoin()) methods
+
+#### Support for multiple (or no) tables in the FROM clause {#orm-from-clause}
+
+Something about how you can now pass in multiple tables for the FROM clause of any `SQLConditionalExpression`, or no from clause at all for an `SQLSelect`.
+
+#### Better support for custom column selections in `DataQuery` {#orm-custom-columns}
+
+Something here about the new [`DataQuery::setAllowCollidingFieldStatements()`](api:SilverStripe\ORM\DataQuery::setAllowCollidingFieldStatements()) method.
+
 ### Other new features
 
 ## API changes
 
-### silverstripe/framework
+### silverstripe/framework {#api-silverstripe-framework}
```

Comment:
Gives a unique anchor ID for this heading to distinguish it from the `silverstripe/framework` heading that will be automatically generated in the actual changelog below this.
Not strictly necessary as part of this PR but it's a tiny change so I probably wouldn't bother with a separate PR.